core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -9168,7 +9173,12 @@ private void throwApplyBlockColumnShuffleFailed(int columnIndex, int columnType,
     }
 
     private void throwDistressException(Throwable cause) {
-        LOG.critical().$("writer error [table=").utf8(tableToken.getTableName()).$(", e=").$((Sinkable) cause).I$();
+        try {
+            Sinkable sinkable = (Sinkable) cause;
+            LOG.critical().$("writer error [table=").utf8(tableToken.getTableName()).$(", e=").$(sinkable).I$();
```

Comment:
i was about to say that must always log table token, because this logging is null-safe, but here are replacement potential problems :)

Reply:
if your point is to log `tableToekn` instead of table name, then it's done in [b865560](https://github.com/questdb/questdb/pull/5634/commits/b865560a292d49198cc3bb9b48f74fd2a04a6866)